pkgs/by-name/hy/hypertimer/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  autoPatchelfHook,
+  godot_4,
+  godot_4-export-templates,
+  libGL,
+  alsa-lib,
+  libpulseaudio,
+  libX11,
+  libXcursor,
+  libXext,
+  libXi,
+  libXrandr,
+  vulkan-loader,
+}:
+
+let
+  godot_version_folder = lib.replaceStrings [ "-" ] [ "." ] godot_4.version;
+  version = "0.2.3";
+in
+stdenv.mkDerivation {
+  name = "hypertimer";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "unfa";
+    repo = "hypertimer";
+    rev = version;
+    hash = "sha256-OFSl58VLjft5BO2QMo0HRYozMI+SBHB+jbVvKQPPnIY=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    autoPatchelfHook
```

Comment:
It may be be better to set `env.NIX_LDFLAGS = "-rpath ${lib.makeLibraryPath runtimeDependencies}"` over this, as it would allows us to link everything while compiling rather than patching the binaries we just built after the fact

Reply:
I used this [pixelorama](https://github.com/NixOS/nixpkgs/blob/9d4887eea159c7bb18b8ff7ed0a84b25b6f9ebae/pkgs/by-name/pi/pixelorama/package.nix) as the starting point for this package.

I'll try the `env.NIXLDFLAGS` that you mentioned. 👍